docs/UserGuide.md

```diff
@@ -156,105 +156,150 @@ Examples:
-    * `reminder 2 weeks` returns chocolate with expiry date of 2020-03-23 only.
-    * `reminder 3 weeks` returns both chocolate with expiry date of 2020-03-23 and banana with expiry date of 2020-03-26.
-  
-### sorting the inventory: `sort`
+If you can't find an item you are looking for, don't worry. Find command will find and display all items 
+whose names contain any of the given keywords, either partially or in full.
 
-Sorts the items from the inventory based on quantity or expiry date.
+<div markdown="block" class="alert alert-info">
 
-Format 1: `sort quantity`
+**:information_source: Notes about the find command:**<br>
 
-* `sort quantity` display the items in the inventory in ascending order of quantity
+* The search is case-insensitive. e.g. `milk` will match `Milk`
+* The order of the keywords does not matter. e.g. `Bread Chocolate` will match `Chocolate Bread`
+* Only the item name is searched.
+* Items matching at least one keyword will be returned. For example, `find Chocolate Milk` will
+  return `Cadbury Chocolate` and `Almond Milk`
+  
+</div>
 
-Format 2: `sort expiryDate`
+* #### 3.6.1. Find items with complete name match
+  Format: `find KEYWORD [MORE_KEYWORDS]`
+  <div markdown="block" class="alert alert-info">
 
-* `sort expiryDate` displays items in the inventory from the earliest expiry date to the latest.
-* Items without expiry date will be pushed to the back of the sorted list.
+  **:information_source: Notes about the command:**<br>
+  
+  * Only full words will be matched e.g. `Chocolate` will not match `Chocolates`<br></br>
+  </div>
+  
+  Examples:
+  * `find Chocolate` returns `chocolate` and `Chocolate Milk`
+  * `find potato chip` returns `Potato Biscuit` and `chocolate chip`
+  
+* #### 3.6.2 Find items with partial name match
+  Format: `find */KEYWORD [MORE_KEYWORDS]`
+  <div markdown="block" class="alert alert-info">
 
-Examples:
+  **:information_source: Notes about the command:**<br>
+  
+  * Partial words will be matched e.g. `Choco` will match `Chocolates`<br></br>
+  
+  </div>
+  
+  Examples:
+  * `find */Burger` returns `CheeseBurger` and `fishburger`
+  * `find */cheese egg` returns `MacAndCheese` and `eggs`
 
-* `find Chocolate` followed by `sort quantity` sorts the items from the result of the `find` command.
-* `list Room 2` followed by `sort expiryDate` sorts the recorded list of items in Room 2 by expiry date.
 
-### Clearing all entries : `clear`
+### 3.7. Viewing expiring items : `reminder`  <a name = "reminder"></a>
 
-Clears all entries from the location book.
+Want to know which items of yours are expiring soon? Then, this is the right command for you.<br>
 
-Format: `clear`
+Format: `reminder number TIME_UNIT`
+<div markdown="block" class="alert alert-info">
 
-### Exiting the program : `exit`
+**:information_source: Notes about the reminder command:**<br>
 
-Exits the program.
+* `TIME_UNIT` must be either `days` or `weeks`.
```

Comment:
will 1 day and 1 week work? or even 1 must be days and weeks?

if 1day and 1week work then that's the best. Else here's my suggestion:
1. change the implementation such that 1 day and 1 week will work.
2. make it extra clear in the user guide (1 more sentence behind it) to emphasise that day and week will not work. :)

Reply:
if I'm not wrong, it only takes in days and weeks as TIME_UNIT. I just went to check the reminderCommandParser. I think we can leave it as it is now and change later if we make changes to the feature.